pkgs/applications/graphics/krita/wrapper.nix

```diff
@@ -0,0 +1,26 @@
+{ callPackage
+, symlinkJoin
+, makeWrapper
+, krita-unwrapped
+, krita-plugin-gmic
+, plugins ? [
+    krita-plugin-gmic
+  ]
+}:
+
+symlinkJoin {
+  name = "krita-${krita-unwrapped.version}";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  paths = [ krita-unwrapped ] ++ plugins;
+
+  postBuild = ''
+    wrapProgram "$out/bin/krita" \
```

Comment:
Are we now double wrapping?😅

Reply:
Yep :laughing:, I seems necessary, otherwise users would need to compile `krita` from source just to install a plugin. Also, `krita-plugin-gmic` requires `krita` as an input.

Reply:
Can we somehow the combine the two wrappers into one and only compile it once? Similar how firefox does it.